packages/dotcom-ui-header/src/components/drawer/topLevelPartials.tsx

```diff
@@ -34,7 +34,10 @@ const Drawer = (props: THeaderProps) => {
         <Search />
         <nav className="o-header__drawer-menu o-header__drawer-menu--primary o-header__drawer-menu--border">
           {editions && <EditionsSwitcher {...editions} />}
-          <ul className="o-header__drawer-menu-list">
+          <ul
+            data-component={props.userIsLoggedIn ? 'drawer-menu--primary__drawer-menu-list' : undefined}
```

Comment:
Question: why do we need the ternary here? Can't we just have the following?
```suggestion
            data-component='drawer-menu--primary__drawer-menu-list'
```

Reply:
> I think the Workspace item should be visible only for logged-in users or am I wrong?
Correct, but I do would like to avoid putting any logic related to our component here. The target data attribute can be present also for anonymous users although not used. In my opinion in this case is better to have less logic in the code and have a useless data attribute on the page for anonymous users. Potentially, it could also be used in the future by someone else script.

Reply:
done. removed the logged in user check